build.zig

```diff
@@ -151,14 +151,15 @@ pub fn build(b: *Build) void {
 
     // geyser reader
     const geyser_reader_exe = b.addExecutable(.{
-        .name = "geyser_reader",
-        .root_source_file = b.path("src/geyser/reader.zig"),
+        .name = "geyser",
+        .root_source_file = b.path("src/geyser/bin.zig"),
```

Comment:
Why not call it `main.zig`? It is the entry-point of the geyser reader.

Reply:
[1dffb78](https://github.com/Syndica/sig/pull/279/commits/1dffb78a0c3d8a3dceb66f4fa5e0de2bfcd41a29)